configmap-kube-root-ca.crt.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+data:
+  ca.crt: |
+    -----BEGIN CERTIFICATE-----
```

Comment:
@abrar-khan007 whatever you are using this cert for you should revoke it immediately